src/Application/Sheets/Commands/DeleteSheet/DeleteSheet.cs

```diff
@@ -21,18 +22,23 @@ public class DeleteSheetCommandHandler(
 
     public async Task<Response> Handle(DeleteSheetCommand request, CancellationToken cancellationToken)
     {
+        Debug.Assert(_user.Id is not null, "User ID should never be null here due to middleware check.");
```

Comment:
?

Reply:
comment for the future: Its a way to avoid the error of _user.Id being null because its impossible as the error message said